documentation/meta/maintenance/elasticsearch_cluster.md

```diff
@@ -336,6 +336,58 @@ etc. The main focus should be understanding the new configuration and
 determining whether it's possible to test it locally. Otherwise, follow the
 steps for deploying a new cluster and switching the live services over to it.
 
+## Node Replacements
+
+Occasionally, the underlying EC2 instance for one of our nodes will undergo a
+scheduled decomission ("retirement") and need to be replaced beforehand.
+
+This procedure describes the process of replacing a single node, but could be
+extrapolated to multiple nodes with relative ease.
+
+If a Data refresh is scheduled during the timeframe of this procedure, it is
+best to pause it for performance reasons.
+
+### Steps
+
+1. Use the Instance ID provided in the retirement notification to find the
+   "Private IPv4 address" of the instance. This value is the "Name" of the
+   ElasticSearch node.
+1. Identify where the node is stored in our Terraform state. This will be useful
+   for manual state changes later in the process. Check each node until you find
+   the matching private IP in the returned state by running
+   `j tf <env> state show "module.<env>-elasticsearch-8-8-2.aws_instance.datanodes[0]"`,
+   `j tf <env> state show "module.<env>-elasticsearch-8-8-2.aws_instance.datanodes[1]"`,
+   and so on for each index. Replace the "<env>" with the correct values. Record
```

Comment:
It's better to avoid using aliases that are so short in documentation and to make this a code block instead of inline for easier copying.

````suggestion
   the matching private IP in the returned state by running
   ```
   just tf <env> state show "module.<env>-elasticsearch-8-8-2.aws_instance.datanodes[0]"
   just tf <env> state show "module.<env>-elasticsearch-8-8-2.aws_instance.datanodes[1]"
   ```
   and so on for each index. Replace the "<env>" with the correct values. Record
````

Also, even avoid using `just` here since it can be messy with the argument and the `""` sometimes. Using Terraform directly has been suggested in the past, but choose whatever you prefer.

````suggestion
   the matching private IP in the returned state by running
   ```
   cd <env>
   terraform state show "module.<env>-elasticsearch-8-8-2.aws_instance.datanodes[0]"
   terraform state show "module.<env>-elasticsearch-8-8-2.aws_instance.datanodes[1]"
   ```
   and so on for each index. Replace the "<env>" with the correct values. Record
````

Reply:
We can (and should) fix the issue with just and quoting by passing arguments through the just `quote` function.

I find `just tf` to be tedious and sometimes unpredictable, and have recommended against using it because of my own confusion with its behaviour. It only sometimes does what I intended, and that frightens me with Terraform, especially direct state manipulations.